homework/src/index.js

```diff
@@ -30,18 +32,113 @@
     return elem;
   }
 
+  const root = document.getElementById('root');
+  const header = createAndAppend('header', root, { id: 'header' });
+  const h3 = createAndAppend('h3', header);
+  h3.innerText = 'HYF Repository';
+  const mainDiv = createAndAppend('div', root, { id: 'main' });
+  const div1 = createAndAppend('div', mainDiv, { id: 'div1' });
+  const div2 = createAndAppend('div', mainDiv, { id: 'div2' });
+  const ul = createAndAppend('ul', div1, { id: 'list-container' });
+  const Contributions = createAndAppend('h3', div2)
+  Contributions.innerText = 'Contributions';
+  const ul2 = createAndAppend('ul', div2, { id: 'list-container' });
+
+  function renderRepositoryInfo(info) {
+    const li = createAndAppend('li', ul);
+    const span1 = createAndAppend('span', li, { id: 'title' });
+    span1.innerText = 'Repository: ';
```

Comment:
You can set the `innerText` directly using `createAndAppend()`:

```js
 createAndAppend('span', li, { id: 'title', text: 'Repository' });
```

If you do it like this you can also get rid of a lot of variables (making you code less complex), because you have many variables here which are only needed to set the `innerText`.